da/celestia/celestia.go

```diff
@@ -59,7 +64,8 @@ func (c *DataAvailabilityLayerClient) Init(
 func (c *DataAvailabilityLayerClient) Start() error {
 	c.logger.Info("starting Celestia Data Availability Layer Client", "baseURL", c.config.BaseURL)
 	var err error
-	c.client, err = cnc.NewClient(c.config.BaseURL, cnc.WithTimeout(c.config.Timeout))
+	ctx := context.Background()
```

Comment:
[nit] no need for a variable, directly do `context.Background()` in `cnc.NewClient(context.Background(), ...`